src/PrestaShopBundle/Security/OAuth2/Repository/NullScopeRepository.php

```diff
@@ -51,6 +51,6 @@ public function finalizeScopes(
         ClientEntityInterface $clientEntity,
         $userIdentifier = null
     ): array {
-        return [];
+        return [new ScopeEntity('hook:read')];
```

Comment:
```suggestion
        return $scopes;
```

Similar suggestion here, we could return the request scopes as is and consider they are all valid for now